src/Neo/Helper.cs

```diff
@@ -29,59 +27,6 @@ public static class Helper
     {
         private static readonly DateTime unixEpoch = new(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);
 
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        private static int BitLen(int w)
```

Comment:
where is this being moved to?

Reply:
no where, It was never used and plus it was set as `private`.